StyleCop.Analyzers/StyleCop.Analyzers/DocumentationRules/SA1600ElementsMustBeDocumented.cs

```diff
@@ -341,6 +363,61 @@ public static void HandleEventFieldDeclaration(SyntaxNodeAnalysisContext context
                     }
                 }
             }
+
+            private static string[] GetDocumentedParameters(SyntaxNodeAnalysisContext context, RecordDeclarationSyntaxWrapper typeDeclarationSyntax, out bool hasInheritdoc)
+            {
+                hasInheritdoc = false;
+
+                var documentation = context.Node.GetDocumentationCommentTriviaSyntax();
+                if (documentation == null)
+                {
+                    return EmptyArray<string>.Instance;
+                }
+
+                if (documentation.Content.GetFirstXmlElement(XmlCommentHelper.InheritdocXmlTag) != null)
+                {
+                    hasInheritdoc = true;
+                    return EmptyArray<string>.Instance;
+                }
+
+                var hasIncludedDocumentation =
+                    documentation.Content.GetFirstXmlElement(XmlCommentHelper.IncludeXmlTag) != null;
+
+                if (hasIncludedDocumentation)
+                {
+                    var declaredSymbol = context.SemanticModel.GetDeclaredSymbol(typeDeclarationSyntax, context.CancellationToken);
+                    var rawDocumentation = declaredSymbol?.GetDocumentationCommentXml(expandIncludes: true, cancellationToken: context.CancellationToken);
+                    var includedDocumentation = XElement.Parse(rawDocumentation ?? "<doc></doc>", LoadOptions.None);
+
+                    if (includedDocumentation.Nodes().OfType<XElement>().Any(element => element.Name == XmlCommentHelper.InheritdocXmlTag))
+                    {
+                        hasInheritdoc = true;
```

Comment:
This code is not executed in tests

Reply:
It handles the case where your XML comment includes an XML file, which itself specifies an `<inheritdoc />`.

I've added a test that should cover this case.